themes/my-test-theme-dark/README.md

```diff
@@ -0,0 +1,12 @@
+# KubeColor Themes
+
+If you want to share your custom theme, please open a Pull Request and:
+- explain in the PR what is the goal of the theme (Ex: simple esthetic, disability related, better efficiency to display informations,...)
+- add a foder for your theme into the `themes` folder, and name it with name of your theme and either the `-dark` or `-light` suffix. Ex: `my-test-theme-dark`.
+- a screen capture of the end result as `.jpg` or `.png`
+- the `color.yaml` file that includes comments with:
+  - your name
+  - your email
+  - the date of the last update
+
+As maintainers we reserve the right to refuse any theme, whatever our reason is. If you don't agree, you are free to host your theme anywhere else. 
```

Comment:
Feels a bit hostile in the phrasing.

I think this is a given and could be skipped


```suggestion
```

Reply:
I see you added a confused smilie, so I just want to explain myself a little. I specifically meant the last part:

> If you don't agree, you are free to host your theme anywhere else.

Like, I agree with the sentiment, but phrasing that is so tricky without sounding nonchalant (and as I don't have any better suggestion in phrasing it) so I find _not saying it_ is better.